src/org/thoughtcrime/securesms/ConversationListItem.java

```diff
@@ -245,7 +245,11 @@ private void setStatusIcons(int visibility, int state, int unreadCount, boolean
     if (visibility==DcChat.DC_CHAT_VISIBILITY_ARCHIVED)
     {
       badgeView.setVisibility(View.VISIBLE);
-      badgeView.setText(R.string.chat_archived_label);
+      if (isContactRequest) {
+        badgeView.setText(getContext().getString(R.string.chat_archived_label) + " " + getContext().getString(R.string.chat_request_label));
```

Comment:
maybe one can do two badge views at some point, however, for now, this is good enough. it is not so much in focus anyway - and saves one view per item :)